index.js

```diff
@@ -58,11 +58,17 @@ async function waitForES({ host = process.env.ES_HOST, maxRetries = 10, msBetwee
 
 
 if (require.main === module) {
-  waitForES({
-    host: process.argv[2],
-    maxRetries: process.argv[3],
-    msBetweenRetries: process.argv[4],
-  })
+  const [
+    host = process.env.ES_HOST,
+    maxRetries = 10,
+    msBetweenRetries = 2000,
+  ] = process.argv.slice(2);
```

Comment:
since you moved the defaults to here, they now only apply to the CLI. If you just import the package and run `waitForEs`, all the values are required

Reply:
ah, didn't see it was exported. Probably won't get to this today or for a while